iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/ManagedIdentityCheck.java

```diff
@@ -224,7 +224,7 @@ private static BiConsumer<CheckContext, ResourceDeclaration> checkManagedIdentit
     return (ctx, resource) -> {
       Optional<Property> identityProperty = resource.resourceProperties().stream().filter(p -> "identity".equals(p.key().value())).findFirst();
       if (identityProperty.isEmpty()) {
-        ctx.reportIssue(resource.name(), OMITTING_IDENTITY_MESSAGE);
+        ctx.reportIssue(resource.symbolicName() != null ? resource.symbolicName() : resource.name(), OMITTING_IDENTITY_MESSAGE);
```

Comment:
I changed it like this to keep the previous behavior. I'm not sure, why here it was done with `name`, as we usually simply highlight the entire resource